server/api/index.js

```diff
@@ -132,6 +132,7 @@ const restrict = (req, res, next) => {
 // Send back a simple JSON error with an HTTP status code
 const errorHandler = (err, req, res, next) => {
   if (err) {
+    console.error(err.stack)
```

Comment:
So every time there is an error when a user accesses the API (including things like attempts at unauthorized access, or missing email addresses), we want to log the entire stack? That seems wrong.

Reply:
Perhaps we should have different error handlers then? There are definitely times when you want the stack trace logged for debug purposes.

Reply:
A quick fix to this would be logging the stack trace when the status is 500 or undefined.